src/app/shared/components/SelectGrid/GridItem/index.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import clsx from 'clsx'
+
+import { usePathname } from 'next/navigation'
+
+import Image from "next/image"
+import Link from "next/link"
+
+type GridItemProps = {
+  entityLogo?: object,
+  entity: {
+    key?: string,
+    primaryColor?: string,
+    title: string,
+  },
+  hideTitle?: boolean,
+}
+
+const GridItem = (props: GridItemProps) => {
+  const { entityLogo, entity, hideTitle } = props || {}
```

Comment:
i think you're going overboard with defaulting to all these empty objects, we always pass props to GridItem so this one at least doesn't need a default